.circleci/config.yml

```diff
@@ -275,6 +303,73 @@ jobs:
       - store_test_results:
           path: libs/SalesforceHybrid/build/outputs/androidTest-results/
 
+  generate-artifacts:
```

Comment:
@bhariharan Linking to test results is not trivial unfortunately.  I could curl and parse this: https://circleci.com/api/v1.1/project/github/brandonpage/SalesforceMobileSDK-Android/latest/artifacts?&branch=ci-experimental&job=generate-artifacts and make the files downloadable but creating and formatting some html isn't high on my priority list.

Reply:
Linking to the workflow head is ideal. Users can drill down into the test runs through the visual interface.